mantle/cmd/ore/gcloud/delete-images.go

```diff
@@ -46,7 +49,15 @@ func runDeleteImage(cmd *cobra.Command, args []string) {
 	for _, name := range args {
 		pending, err := api.DeleteImage(name)
 		if err != nil {
-			fmt.Fprintf(os.Stderr, "%v\n", err)
+			// Skip on NotFound error only if allowMissing flag is set to True
+			if gErr, ok := err.(*googleapi.Error); ok && allowMissing {
+				// Check if the error code is 404 (Not Found)
+				if gErr.Code == 404 {
```

Comment:
`&& allowMissing` check should be here instead

Reply:
Alternatively, do something similar what you did in the AWS one and only downcast in the `allow_missing` case.